features/facebookImport/src/model/analyses/ministories/activities-analysis.js

```diff
@@ -57,7 +57,7 @@ export default class ActivitiesAnalysis extends RootAnalysis {
             groupedActivities.total++;
         });
 
-        dataAccount.analyses.totalEvents =
+        dataAccount.processedData.totalEvents =
```

Comment:
Also, you're using the "object" style here for assignment, while you're using the "array" style somewhere else via square brackets.

Reply:
That's customary to do when you know the property, hard coding what's in the brackets would be weird: `dataAccount["processedData"]`

Reply:
But it's used that way elsewhere...